test/Safe/Safe.t.sol

```diff
@@ -0,0 +1,145 @@
+pragma solidity ^0.8.0;
+
+import "src/TestBase.sol";
+
+import {
+    SAFE_FACTORY_BYTECODE,
+    SAFE_ACCOUNT_BYTECODE,
+    SAFE_4337_MODULE_BYTECODE,
+    SAFE_FACTORY,
+    SAFE_4337_MODULE,
+    ADD_MODULES_LIB_BYTECODE,
+    ADD_MODULES_LIB,
+    SAFE_ACCOUNT,
+    SafeProxyFactory,
+    Safe4337Module,
+    Safe,
+    AddModulesLib
+} from "./SafeArtifacts.sol";
+
+contract SoladyERC4337Test is AAGasProfileBase {
+    SafeProxyFactory factory;
+
+    function setUp() external {
+        initializeTest("Safe");
+        factory = SafeProxyFactory(SAFE_FACTORY);
+        vm.etch(SAFE_FACTORY, SAFE_FACTORY_BYTECODE);
+        vm.etch(SAFE_ACCOUNT, SAFE_ACCOUNT_BYTECODE);
+        vm.etch(SAFE_4337_MODULE, SAFE_4337_MODULE_BYTECODE);
+        vm.etch(ADD_MODULES_LIB, ADD_MODULES_LIB_BYTECODE);
+        setAccount();
+    }
+
+    function fillData(address _to, uint256 _value, bytes memory _data) internal view override returns (bytes memory) {
+        return abi.encodeWithSelector(Safe4337Module.executeUserOp.selector, _to, _value, _data, 0);
+    }
+
+    function getSignature(UserOperation memory _op) internal view override returns (bytes memory) {
+        (bytes memory operation, uint48 validAfter, uint48 validUntil) = _getSafeOp(_op);
+        (uint8 v, bytes32 r, bytes32 s) = vm.sign(key, keccak256(operation));
+        bytes memory signature = abi.encodePacked(r, s, v);
+        return abi.encodePacked(validAfter, validUntil, signature);
+    }
+
+    function createAccount(address _owner) internal override {
+        (bool success, bytes memory data) = address(factory).call(
+            abi.encodeWithSelector(factory.createProxyWithNonce.selector, SAFE_ACCOUNT, _initializerCalldata(_owner), 0)
```

Comment:
@leekt is this `_initializerCalldata` going to be captured in the benchmark? Seems not but want to double check

Reply:
No it is not going to be captured, aa-benchmark only check for balance change of the wallet and the deposit change on the entrypoint so data packing gas will not be added in the benchmarks :)